apps/services/models.py

```
@@ -0,0 +1,67 @@
+
+    class Meta:
+        verbose_name = _('Service')
+        verbose_name_plural = _('Services')
+        db_table = 'services'
+        ordering = ['name']
+        indexes = [
+            models.Index(fields=['vendor', 'is_active'], name='services_vendor_active'),
+        ]
+        unique_together = ('vendor', 'name')
+
+    def __str__(self):
```

Ruff (0.14.0)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)